src/wp-includes/media.php

```diff
@@ -5753,11 +5781,39 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 	 * If the element is in the viewport (`true`), potentially add
 	 * `fetchpriority` with a value of "high". Otherwise, i.e. if the element
 	 * is not not in the viewport (`false`) or it is unknown (`null`), add
-	 * `loading` with a value of "lazy".
+	 * `loading` with a value of "lazy" and set `decoding` with the value of
+	 * "async".
 	 */
 	if ( $maybe_in_viewport ) {
 		$loading_attrs = wp_maybe_add_fetchpriority_high_attr( $loading_attrs, $tag_name, $attr );
 	} else {
+		// Only add `decoding` attribute if the feature is enabled.
+		if ( wp_decoding_enabled( $tag_name, $context ) ) {
+			$decoding = array_key_exists( 'decoding', $attr ) ? $attr['decoding'] : 'async';
+
+			/**
+			 * Filters the `decoding` attribute value to add to an image. Default `async`.
+			 *
+			 * Returning a falsy value will omit the attribute.
+			 *
+			 * @param string|false|null $value    The `decoding` attribute value. Returning a falsy value
+			 *                                    will result in the attribute being omitted for the image.
+			 *                                    Otherwise, it may be: 'async' (default), 'sync', or 'auto'.
+			 * @param string            $tag_name The tag name of the element.
+			 * @param string            $context  Additional context about how the function was called
+			 *                                    or where the img tag is.
+			 *
+			 * @since 6.4.0
```

Comment:
Per [PHP Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters) `@since` annotation prior to `@param`

Reply:
Thanks for the reference ;)